src/main/kotlin/org/wycliffeassociates/tstudio2rc/FileUtils.kt

```diff
@@ -0,0 +1,60 @@
+package org.wycliffeassociates.tstudio2rc
+
+import com.fasterxml.jackson.core.JsonFactory
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.wycliffeassociates.tstudio2rc.serializable.BookVersification
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+import java.util.zip.ZipOutputStream
+
+internal fun loadJsonObject(path: String): Map<String, Any> {
```

Comment:
You'd probably rather work with this as a JsonNode than Map<String, Any>, and use readTree(File(path)) instead